apis/workloads/v1alpha1/lifecycle_recorder.go

```diff
@@ -0,0 +1,147 @@
+package v1alpha1
+
+import (
+	"context"
+	"time"
+
+	"github.com/go-logr/logr"
+	"github.com/gocardless/theatre/v3/pkg/workloads/console/events"
+)
+
+func CommonEventFromConsole(ctx string, eventKind events.EventKind, csl *Console) events.CommonEvent {
+	return events.CommonEvent{
+		Version:    "v1alpha1",
+		Kind:       events.KindConsole,
+		Event:      events.EventRequest,
```

Comment:
Should this be `eventKind`?

Reply:
Thanks for catching this. Updating this.